libc/fuzzing/stdlib/heap_sort_fuzz.cpp

```diff
@@ -0,0 +1,53 @@
+//===-- heap_sort_fuzz.cpp ----------------------------------------------------===//
```

Comment:
```suggestion
//===-- heap_sort_fuzz.cpp ------------------------------------------------===//
```

Tips:

- Type the title in replace mode.
  - `Insert` key on editors without esoteric keybinds.
  - `Shift+R` on Vim/Neovim.
- Configure your editor to display a column limit at 80 columns.
  - `vim.opt.colorcolumn = "80"` in Lua on Neovim.
  - `set colorcolumn=80` in Vimscript on Vim/Neovim.
  - `"editor.rulers": [80],` in JSON on VS Code.
  - **Settings** -> **Editor** -> **Code Style** -> either **Hard wrap at** or **Visual guides** on JetBrains IntelliJ-based IDEs.